src/StatCache.cc

```diff
@@ -59,7 +59,7 @@ int StatCache::stat(const char *path, struct stat *st)
 		}
 		statMap.erase(iter);                            // Remove stale entry
 	}
-	CacheEntry entry{};                               // Make a new entry
+	CacheEntry entry;                                 // Make a new entry
```

Comment:
thx - that's an old gcc with not-yet-full C++ support. Anyway, I agree that initializers are not strictly necessary here.